### ISSUE TITLE
UniqueArguments omitted when serializing a webhook event

### ISSUE THREAD

**Issue by Jericho** [Jericho/StrongGrid#493]
@elmalakai [pointed out](https://github.com/Jericho/StrongGrid/issues/492#issuecomment-1572670743) that `UniqueArguments` is omitted when serializing a webhook event.

**Comment by Jericho** [comment]
@elmalakai here's a beta version of the next package that will properly serialize uniqueargs: https://www.myget.org/feed/jericho/package/nuget/StrongGrid/0.101.0-beta0005

The only concession we have to make is that unique arguments with a numerical value will be serialized as a string like so:
```
"reseller_id": "1234"
```

which is not 100% same as the original JSON received from SendGrid but it's the best we can do due to the fact that unknown values must be cast as string in order to be added to the `Dictionary<string, string>`.

**Comment by Jericho** [comment]
Thanks for confirming.

**Comment by elmalakai** [comment]
Looks like you had an bug in [0.101.0-beta0005](https://www.myget.org/feed/jericho/package/nuget/StrongGrid/0.101.0-beta0005) but I think you've fixed it. I tried to write a test case for it and it's working. Specifically, the UniqueArguments were being double wrapped in double quotes when deserialized:
```
? events[0].UniqueArguments["environment"]
"\"Production\""
```
Tried to write a test case for it, but I think you beat me to it (Webhook_includes_unknow_property).
